src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -28,20 +28,21 @@ public TorpedoStore(int numberOfTorpedos){
     }
   }
 
+  private final Random generator = new Random();
```

Comment:
What about renaming `generator` to `randomGenerator`

```suggestion
  private final Random randomGenerator = new Random();
```

Reply:
I do not agree with you.....